tests/drivers/bc12/boards/native_posix.conf

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_EMUL=y
+CONFIG_GPIO=y
+CONFIG_I2C=y
+CONFIG_USB_BC12=y
+CONFIG_EMUL_BC12_PI3USB9201=y
```

Comment:
Maybe it makes sense to be explicit in the test, but shouldn't this be automatic from the devicetree?

Reply:
That wasn't the current pattern for emulators, but it is a good idea.  Note that the emulator config must also be guarded with CONFIG_ZTEST because the emulator and driver share the same compatible string.